### ISSUE TITLE
Power surge balance

### ISSUE THREAD

**Issue by xindium** [mlenser/kryx-rpg-issues#111]
Specifically concerning Death, Blood and Unholy Themes, which read as follows:

Death. Necrotic energies swirl around you, draining the life of nearby creatures. Up to 3 creatures of your choice that you can see within 9 meters of you take necrotic damage equal to twice the mana expended. Alternatively, you can siphon the life of nearby creatures. Up to 3 creatures of your choice that you can see within 9 meters of you take necrotic damage equal to the mana expended. You regain hit points equal to half the total amount of necrotic damage dealt.

Blood. You pull the life force from nearby creatures. Up to 3 creatures of your choice that you can see within 9 meters of you take necrotic damage equal to the mana expended. You regain hit points equal to half the total amount of necrotic damage dealt.

Unholy. Unholy energy surrounds you. Up to 3 creatures of your choice that you can see within 9 meters of you take necrotic damage equal to twice the mana expended.

In other words, death is just freely choosing between unholy and blood, making it an obviously superior choice to the two (as bloodragers choose a default theme)

**Comment by mlenser** [comment]
Death as a theme encompases the death and undeath ideas, so both are represented here. Similarly Life Drain and Inflict wounds exist in Death while Blood's Avasculate isn't as central to the theme as Blood Drain is.

Some options are better based on their theme. I don't think it makes sense to add the Avasculate flavor to Power Surge under blood.

**Comment by mlenser** [comment]
Closing this as certain themes simply have more flavor so more versatility. If you think things should change please follow up.
